NEWS.md

```diff
@@ -1,5 +1,7 @@
 # vroom (development version)
 
+* Add a section to getting started vignette to show to how handle multiple files in a zip (@meta00, #125) 
```

Comment:
You don't need a news bullet here, since it's just a documentation change. Can you please remove it?

Reply:
done in commit https://github.com/r-lib/vroom/pull/156/commits/06c8cbfd2c388939ac61395ecc5e1ad0b2a2a2d3